### DISCUSSION TITLE
Trakt Ids not uniq ?

### DISCUSSION THREAD

**Discussion by MoonArrow** [trakt/trakt-api#377] [Q&A] [answered]
Hi,

I am using Trakt.tv with pleasure since many years now. For some statistical purpose, I started to write a small script to take my tvshows/episodes/movies seen history. I store the relevant information to avoid calling Trakt API too often.

During the process, I found 2 episodes with the same ID (2037955) Lucifer S01E08 and 9-1-1 S03E11. The id I use is the id in ids structure when calling https://api.trakt.tv/shows/id/seasons/season/episodes/episode

I always assumed that this id was uniq. Am I wrong or have I just found an incoherence in database ?

If this Id is not uniq by design, is there a uniq ID I can use on the movies/tvshows/episodes to rely on ?

Thanks
M.

**Comment by tysonkerridge** [comment, marked as answer]
Not sure if I'm looking at a different set of IDs or endpoint usage to you but it looks like you might be confusing/mixing two different source IDs?
Looks like the Lucifer episode has `trakt` ID 2037955 but the 9-1-1 episode has `trakt` ID 3875399, whereas it does have "the same" ID of 2037955 but for `tmdb`.

`GET https://api.trakt.tv/shows/lucifer/seasons/1?extended=full`
```json
        "season": 1,
        "number": 8,
        "title": "Et Tu, Doctor?",
        "ids": {
            "trakt": 2037955,
            "tvdb": 5531464,
            "imdb": "tt4952854",
            "tmdb": 1165518,
            "tvrage": 1065885231
        },
```

`GET https://api.trakt.tv/shows/9-1-1/seasons/3?extended=full`
```json
        "season": 3,
        "number": 11,
        "title": "Seize the day",
        "ids": {
            "trakt": 3875399,
            "tvdb": 7489097,
            "imdb": "tt11713522",
            "tmdb": 2037955,
            "tvrage": null
        },
```
